ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -336,6 +336,66 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
         return Eigen::Map<const Eigen::RowVectorXd>(N.data(), N.size());
     }
 
+    Eigen::Vector3d getFlux(MathLib::Point3d const& pnt_local_coords,
+                            double const t,
+                            std::vector<double> const& local_x) const override
+    {
+        // eval dNdx and invJ at given point
```

Comment:
A latter comment says that only dNdx is needed...

Reply:
Fixes comment.